views/apps/create.html

```diff
@@ -41,7 +41,7 @@ <h2 class="font-bold text-2xl font-headline mb-2">{{.Name}}</h2>
       class="inline-flex bg-purple-700 border border-warmGray cursor-pointer dark:bg-surface-02dp dark:border-neutral-800 dark:hover:bg-surface-16dp dark:text-neutral-200 duration-150 focus-visible:ring-2 focus-visible:ring-offset-2 focus-visible:ring-orange-bitcoin focus:outline-none font-medium hover:bg-purple-900 items-center justify-center px-5 py-3 rounded-md shadow text-white transition"
       href="#"
     >
-      <i class="fas fa-eye mr-3" aria-hidden="true"></i>
+      <img src="public/images/visible.svg" width="24" height="24" />
```

Comment:
@bumi any idea how we can inline the SVG here? Cannot change its colour unless it's inlined. currently the image has a black `fill`:

<img width="293" alt="image" src="https://user-images.githubusercontent.com/100707419/234395662-8e66414c-8b6d-46a3-a1e0-3d708d490a9b.png">

Reply:
!? https://stackoverflow.com/a/40639621

Reply:
c&p it in the HTML? 

imo don't even try to think in helpers and such

Reply:
i fixed it by changing the colour in the SVG file. Will c&p if we ever need the eye in a different colour later.